modules/golang/golang

```diff
@@ -1,5 +1,11 @@
 compdef _golang go
 
+function _deps(){
+	app="go"
+
+	$(hash $app 2>/dev/null) || $(myzsh error "Couldn't find application ${app}" && return 1)
```

Comment:
$() means run that in a subshell, and return the value, () means just run that in a subshell. Is a subshell even needed? Can this be just `hash $app 2>/dev/null || myzsh error "Couldn't find …" && return 1`?